tests/std/tests/P1208R6_source_location/test.cpp

```diff
@@ -78,7 +90,13 @@ constexpr void different_constructor_test() {
     const s x{1};
     assert(x.loc.line() == s_int_line);
     assert(x.loc.column() == 5);
+#if defined(__clang__) || defined(__EDG__) // TRANSITION, DevCom-10199227 and LLVM-58951
     assert(x.loc.function_name() == "s"sv);
+#elif defined(_M_IX86) // ^^^ workaround / no workaround vvv
+    assert(x.loc.function_name() == "__thiscall s::s(int)"sv);
+#else // _M_IX86

```

Comment:
I guess this should be
```suggestion
#else // ^^^ _M_IX86 // !_M_IX86 vvv
```

Reply:
same below